hazelcast/src/main/java/com/hazelcast/client/impl/protocol/codec/DynamicConfigAddDataLinkConfigCodec.java

```diff
@@ -35,10 +35,10 @@
 
 /**
  * Adds a data link configuration.
- * If an data link configuration with the given {@code name} already exists, then
+ * If a data link configuration with the given {@code name} already exists, then
```

Comment:
this change does not belong to related PR in client-protocol. In should not be here.

Reply:
Yes, it should've been included in the related OS PR, for which I am not responsible. I think creating a PR to fix an auto-generated file and including that fix in an unrelated PR are equally ugly.

Reply:
IMO fixing it in separate PR would be better, but if this is a conscious decision, then it is fine.